app/api_mocks/party_service_mock.py

```diff
@@ -22,7 +23,7 @@ def get_user_details(self, uuid):
         try:
             return Response(response=json.dumps(self._respondent_ids[uuid]), status=200, mimetype="text/html")
         except KeyError:
-            logger.debug('User ID %s not in mock party service', uuid)
+            logger.debug('User ID not in mock party service', uuid=uuid)
```

Comment:
We should make sure whatever we're calling this (and other things) are consistent across services as well. I know in frontstage we're call this party_id most of the time